org.lflang/src/org/lflang/validation/AttributeSpec.java

```diff
@@ -206,13 +204,17 @@ enum AttrParamType {
     static {
         // @label("value")
         ATTRIBUTE_SPECS_BY_NAME.put("label", new AttributeSpec(
-            List.of(new AttrParamSpec(AttributeSpec.VALUE_ATTR, AttrParamType.STRING, null))
+            List.of(new AttrParamSpec(VALUE_ATTR, AttrParamType.STRING, true))
         ));
         // @sparse
         ATTRIBUTE_SPECS_BY_NAME.put("sparse", new AttributeSpec(null));
         // @icon("value")
         ATTRIBUTE_SPECS_BY_NAME.put("icon", new AttributeSpec(
-            List.of(new AttrParamSpec(AttributeSpec.VALUE_ATTR, AttrParamType.STRING, null))
+            List.of(new AttrParamSpec(VALUE_ATTR, AttrParamType.STRING, true))
+        ));
+        // @enclave(indivdual=boolean)
+        ATTRIBUTE_SPECS_BY_NAME.put("enclave", new AttributeSpec(
```

Comment:
Since we don't manage default values centrally at the moment, I don't think it should. I agree that we should provide a better infrastructure for attribute parameters, but this should be done in another PR.

Reply:
It looks like the term "enclave" is getting established. Did we decide the terminology collision with Intel SGX doesn't matter?